pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -1594,15 +1593,23 @@ func verifyVolumeAttachedToNode(
 	expectedAttachState cache.AttachState,
 	asw cache.ActualStateOfWorld,
 ) {
-	attachState := asw.GetAttachState(volumeName, nodeName)
```

Comment:
this is changing a synchronous check to an asynchronous one. is that correct (the thing we were waiting for was actually asynchronous) or is this poll masking a bug in something that *should* be synchronously true?